src/tests/AttributesManagersTest.cpp

```diff
@@ -355,15 +354,15 @@ class AttributesManagersTest : public testing::Test {
       const std::string& str_val,
       bool bool_val,
       int int_val,
-      float float_val,
+      double double_val,
       Magnum::Vector3 vec_val,
       Magnum::Quaternion quat_val) {
     // user defined attributes from light instance
     ASSERT_NE(nullptr, userConfig);
     ASSERT_EQ(userConfig->getString("user_string"), str_val);
     ASSERT_EQ(userConfig->getBool("user_bool"), bool_val);
     ASSERT_EQ(userConfig->getInt("user_int"), int_val);
-    ASSERT_EQ(userConfig->getFloat("user_float"), float_val);
+    ASSERT_EQ(userConfig->getDouble("user_double"), double_val);
```

Comment:
Why doesn't the new API have 32-bit floats anymore? Those are used for physics, for the transformations, for most of everything.

Reply:
Now, there's no need to distinguish between floats and doubles (Same map and all that) so I will reintroduce them.  

But at the time when I made the above change, I was using separate maps for floats and doubles, and since there's no way to distinguish between doubles and floats in python, I found this to be a potential problem. Granted, there's no way to distinguish them in json either (rapidjson just does a cast from double to float if you want something as a float) so i thought there was minimal loss.

Reply:
Due to both the python and the json not distinguishing between floats and doubles, I am going to keep the Configuration to only support doubles for now, unless someone has some other suggestions.